api/test/wfl/integration/modules/covid_test.clj

```diff
@@ -2,18 +2,61 @@
   (:require [clojure.test :refer :all]
             [clojure.test :as clj-test]
             [wfl.tools.fixtures :as fixtures]
-            [wfl.jdbc :as jdbc]
-            [wfl.api.workloads :as workloads]))
+            [wfl.tools.workloads :as workloads]))
 
 (clj-test/use-fixtures :once fixtures/temporary-postgresql-database)
 
-(deftest test-create-covid-workflow
-  (jdbc/with-db-transaction [tx (fixtures/testing-db-config)]
-                            (workloads/create-workload! tx {:pipeline "Sarscov2IlluminaFull"
-                                                            :source {:name "Terra DataRepo"}})
-                            )
-  )
+(def ^:private testing-dataset "ff6e2b40-6497-4340-8947-2f52a658f561")
+(def ^:private testing-workspace "general-dev-billing-account/test-snapshots")
+(def ^:private testing-method-configuration "pathogen-genomic-surveillance/sarscov2_illumina_full")
+
+(defn ^:private make-covid-workload-request []
+  (-> (workloads/covid-workload-request testing-dataset
+                                        testing-method-configuration
+                                        testing-workspace)
+      (assoc :creator @workloads/email)))
+
+(deftest test-create-covid-workload
+  (testing "That COVID workload-request can pass verification"
```

Comment:
I like seeing your test cases called out like this.

Reply:
I prefer only using `testing` if you're gonna have multiple different tests within one test point. If it's a single thing, call your test something more descriptive?

Reply:
Can you elaborate please? I am confused by your comment. This test does test several things.

Reply:
I try to avoid multiple `testings` and prefer to write separate `deftest`s. The reason is that each test point can be run in isolation. This also has the advantage in that if one fails, it can be marked `^:excluded` until it's fixed  - i.e. the test runner won't run tests annotated with `^:excluded`. If you have one `deftest` and lots of `testing` expressions, you can't do this.

I make an exception when there's some shared resource that's expensive to create and needs to be shared across test points and used in a specific order (see `rawls-test/test-snapshot-references` in `develop` for an example). It follows that I don't assume the order in which tests are run and don't drool state across tests.

Reply:
Applied to this PR, I'd prefer if you used `deftest` instead of `testing` for `test-create-covid-workload` because each test point is independent (they don't share state).

Reply:
Thanks - this explanation helped a lot!